data/data.json

```diff
@@ -12647,14 +12647,12 @@
    "ingressWebhook": "rancher/mirrored-ingress-nginx-kube-webhook-certgen:v20231011-8b53cabe0",
    "metricsServer": "rancher/mirrored-metrics-server:v0.6.3",
    "windowsPodInfraContainer": "rancher/mirrored-pause:3.7",
-   "aciCniDeployContainer": "noiro/cnideploy:5.2.7.1.81c2369",
-   "aciHostContainer": "noiro/aci-containers-host:5.2.7.1.81c2369",
-   "aciOpflexContainer": "noiro/opflex:5.2.7.1.81c2369",
-   "aciMcastContainer": "noiro/opflex:5.2.7.1.81c2369",
-   "aciOvsContainer": "noiro/openvswitch:5.2.7.1.81c2369",
-   "aciControllerContainer": "noiro/aci-containers-controller:5.2.7.1.81c2369",
-   "aciGbpServerContainer": "noiro/gbp-server:5.2.7.1.81c2369",
-   "aciOpflexServerContainer": "noiro/opflex-server:5.2.7.1.81c2369"
```

Comment:
The images `aciGbpServerContainer` and `aciOpflexServerContainer` are not required anymore in `aci 6.0.3`?

Reply:
based on https://github.com/rancher/kontainer-driver-metadata/pull/1219/commits/cf48b3cb670a590028170daf5bafcf5416d7642b